packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.types.ts

```diff
@@ -86,6 +86,12 @@ export interface IVerticalBarChartProps extends ICartesianChartProps {
    */
   hideLabels?: boolean;
 
+  /**
+   * Prop to enable support for negative values
+   * @default false
+   */
+  supportNegativeYValues?: boolean;
```

Comment:
>supportNegativeYValues [](http://example.com/codeflow?start=2&length=22)

This is not needed. Adding the prop in ICartesianChartProps is sufficient

Reply:
The thought was that in similar way, we can define other props to enable control in different quadrants. Should I remove it for now?

Reply:
Not needed right now. We will add it on need basis.

Reply:
Sure